.build/release_artifacts.sh

```diff
@@ -58,11 +58,27 @@ done
 
 tag_latest() {
     local new_version=$1
+    local minor_version=${new_version%.*}
```

Comment:
nit: let's make this variable `major_minor_version`

Reply:
Or `version_without_patch`.

Reply:
Also a link to SO or a bash docs page would help if we ever want to change this.